scripts/install.sh

```diff
@@ -13,6 +13,9 @@ printf 'Y\n' | apt-get install -y maven
 printf 'Y\n' | apt-get install git
 printf 'Y\n' | apt-get install npm
 
+#install bro
+sudo apt-get install bro
```

Comment:
The above line (line-17) belongs to tool-install.sh. I know I suggested otherwise in the previous code review. My apologies for the mistake.

Reply:
You can move the command to the last line of tool-install.sh